pkgs/mplus_bitmap_font/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  pkgs,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "mplus_bitmap_font";
+  version = "2.2.4";
+
+  src = fetchurl {
+    url = "https://github.com/coz-m/MPLUS_FONTS/raw/c47fd4ff0a604d1517625a0f3d67e6d64e12d585/obsolete/mplus_bitmap_fonts-2.2.4.tar.gz";
```

Comment:
Switching to https://github.com/xmisao/xfonts-umplus (https://www.xmisao.com/2013/08/17/mplus-unicode-font.html) might be easy